packages/ember-glimmer/lib/modifiers/action.js

```diff
@@ -176,12 +176,14 @@ export default class ActionModifierManager {
       if (actionNameRef[INVOKE]) {
         actionName = actionNameRef;
       } else {
+        let actionLabel = actionNameRef._propertyKey;
```

Comment:
I'm hoping somebody might point out a better way to do this 😄 

There is another area that is doing something similar https://github.com/emberjs/ember.js/pull/13684#discussion-diff-67275895